ui/.vercel/build.sh

```diff
@@ -0,0 +1,6 @@
+STORYBOOK_LINK=true ember build
+mkdir -p ui-dist/ui
+mv dist/* ui-dist/ui/
+
+yarn build-storybook
+mv storybook-static ui-dist/storybook/
```

Comment:
I totally agree that this should live in the repo rather than the Vercel configuration UI 🙇